utils/format_submission.R

```diff
@@ -0,0 +1,122 @@
+
+# Packages ----------------------------------------------------------------
+
+require(here)
+require(dplyr)
+require(lubridate)
+
+## Arguments
+# loc: path to folder
+# loc_name: name of region
+# forecast_date: date forecast is made
+# forecast_adjustment: 
+format_forecast <- function(loc = NULL, loc_name = NULL,
```

Comment:
the forecast adjustment needs to be set to be the median total delay (so report + incubation). We also do this internally in EpiNow now but this would need hookinh up (I tried this for the UK - broke everything and reverted one night at 11)